internal/cmd/flink/command_compute_pool_update.go

```diff
@@ -79,6 +78,14 @@ func (c *command) computePoolUpdate(cmd *cobra.Command, args []string) error {
 		update.Spec.MaxCfu = flinkv2.PtrInt32(cfu)
 	}
 
+	name, err := cmd.Flags().GetString("name")
```

Comment:
Do you mind adding the following to the beginning of the function? We want to make sure at least one of `--name` or `--cfu` is passed.
```
if err := errors.CheckNoUpdate(cmd.Flags(), "name", "cfu"); err != nil {
	return err
}
```